app/models/validations/tenancy_validations.rb

```diff
@@ -40,6 +40,7 @@ def validate_other_tenancy_type(record)
   end
 
   def validate_joint_tenancy(record)
+    return if record.joint == 3
```

Comment:
return unless joint is yes and remove the check for != 2 on line 46?

Reply:
ok i think i get you, i've tweaked

Reply:
Do we need this return at all now?

Reply:
you are correct, removing